cves/CVE-2004-0747.yml

```diff
@@ -167,17 +187,19 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      User supplied strings through config files were not being validated by length. This caused the overflow 
+      and subsequent gain of privledges.
   security_by_obscurity:
     applies:
     note:
   serial_killer:
     applies:
     note:
   environment_variables:
-    applies:
-    note:
+    applies: true
+    note: Environment variable expansion is what caused the buffer overflows, so this applies.
```

Comment:
Just a formatting tweak, no need to say 'so this applies' when you say 'applies: true' right above it

Reply:
You're right 👍